pkgs/development/tools/lemminx/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl, unzip, zlib, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "lemminx";
+  version = "0.22.0";
+
+  src = fetchurl {
+    url = "https://github.com/redhat-developer/vscode-xml/releases/download/${version}/lemminx-linux.zip";
+    sha256 = "sha256-cbRqdgYHHClnlTWgA1vBdbAT6EzD7ygvfM9Zppl5czo=";
+  };
+
+  nativeBuildInputs = [ unzip autoPatchelfHook ];
+  buildInputs = [ zlib ];
+
+  unpackPhase = ''
+    unzip $src
```

Comment:
I think this happens automatically if you put `unzip` into `nativeBuildInputs`. can you check if this works without this phase?